packages/@glimmer/component/src/tracked.ts

```diff
@@ -12,6 +12,55 @@ import { dict, Dict } from "@glimmer/util";
  * Marking a property as tracked means that when that property changes,
  * a rerender of the component is scheduled so the template is kept up to date.
  *
+ * There are two usages for the `@tracked` decorator, shown below.
+ *
+ * @example No dependencies
+ *
+ * If you don't pass an argument to `@tracked`, only changes to that property
+ * will be tracked:
+ *
+ * ```typescript
+ * import Component, { tracked } from '@glimmer/component';
+ *
+ * export default class MyComponent extends Component {
+ *    @tracked
+ *    remainingApples = 10
+ * }
+ * ```
+ *
+ * When something changes the component's `remaining` property, the rerender
+ * will be scheduled.
+ *
+ * @example Dependents
+ *
+ * In the case that you have a computed property that depends on another
+ * property, you want to track both so that when one of them changes a
+ * rerender is scheduled.
+ *
+ * In the following example we have three properties, `totalApples`,
+ * `eatenApples`, and `remainingApples`.
+ *
+ *
+ * ```typescript
+ * import Component, { tracked } from '@glimmer/component';
+ *
+ * export default class MyComponent extends Component {
+ *   totalApples = 100
+ *
+ *   @tracked
+ *   eatenApples = 0
+ *
+ *   @tracked('totalApples', 'eatenApples')
```

Comment:
this seems strange. why is `totalApples` mentioned as a dependency, but `totalApples` above is not marked as `@tracked`?

Reply:
I think `totalApples` can be removed from the dependencies.

Reply:
we then need to explicitly explain it though that it does not have to be mentioned because it itself is not `@tracked` and probably even why it is not `@tracked`